test/tunnel.spec.js

```diff
@@ -149,4 +157,10 @@ describe('screener-runner/src/tunnel', function() {
       expect(ngrokMock.disconnect.called).to.equal(true);
     });
   });
+
+  describe('Sauce Connect Launcher', function() {
+    it('should be downloaded after screener runner is installed', function() {
+      expect(path.resolve(`./node_modules/sauce-connect-launcher/sc/sc-${SAUCE_CONNECT_LAUNCHER_VERSION}-${PLATFORM}/bin/sc`)).to.be.a.file('Didn\'t download sauce connect launcher correctly');
+    });
+  });
```

Comment:
@loyalchow @xwa130saucer this feels more like a functional test, what do you think on spliting tests into different folders
```
test
|_ unit
|_ functional
```

Reply:
I'm OK with both, how do you think? @loyalchow

Reply:
I think this is fine with the current tests. But let's separate into another spec file - `install.spec.js` or something like that.

Reply:
Addressed